website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -35,7 +35,7 @@ programming guide, take a look at the
 {{< language-switcher java py go typescript yaml >}}
 
 {{< paragraph class="language-py" >}}
-The Python SDK supports Python 3.8, 3.9, 3.10, and 3.11.
+The Python SDK supports Python 3.8, 3.9, 3.10, 3.11, and 3.12.
```

Comment:
Should we also drop 3.8 from this list? According to the `tox.ini` [file](https://github.com/apache/beam/blob/master/sdks/python/tox.ini#L20), 3.8 is no longer tested.